odoo/addons/base/tests/test_configmanager.py

```diff
@@ -438,12 +438,15 @@ def test_05_repeat_parse_config(self):
 
     def test_06_cli(self):
         with file_open('base/tests/config/cli') as file:
-            with self.assertLogs('odoo.tools.config', 'WARNING') as capture:
+            with self.assertLogs('odoo.tools.config', 'WARNING') as capture, self.assertLogs('py.warnings', 'WARNING') as capture_2:
                 self.config._parse_config(file.read().split())
         self.assertEqual(capture.output, [
             "WARNING:odoo.tools.config:option --without-demo: since 19.0, invalid boolean value: 'rigolo', assume True",
             "WARNING:odoo.tools.config:test file '/tmp/file-file' cannot be found",
         ])
+        self.assertEqual([
+            Like("... DeprecationWarning: The without-demo option should be used as a flag ..."),
+        ], capture_2.output)
```

Comment:
there already is a warning:

> WARNING:odoo.tools.config:option --without-demo: since 19.0, invalid boolean value: 'rigolo', assume True

why adding another one?